src/rules/selector-attribute-quotes/index.js

```diff
@@ -0,0 +1,70 @@
+import {
+  isStandardSyntaxRule,
+  isStandardSyntaxSelector,
+  parseSelector,
+  report,
+  ruleMessages,
+  validateOptions,
+} from "../../utils"
+
+export const ruleName = "selector-attribute-quotes"
+
+export const messages = ruleMessages(ruleName, {
+  expectedQuotes: "Expected quotes",
+  rejectedQuotes: "Unexpected quotes",
+})
+
+function isQuotesExist(str) {
+  if (str[0] && str[str.length - 1] === "\""
+    || str[0] && str[str.length - 1] === "'"
+  ) { return true }
+
+  return false
+}
+
+export default function (expectation) {
+  return (root, result) => {
+    const validOptions = validateOptions(result, ruleName, {
+      actual: expectation,
+      possible: [
+        "always",
+        "never",
+      ],
+    })
+    if (!validOptions) { return }
+
+    root.walkRules(rule => {
+      if (!isStandardSyntaxRule(rule)) { return }
+      if (rule.selector.indexOf("[") === -1
+        || rule.selector.indexOf("=") === -1
+      ) { return }
+
+      parseSelector(rule.selector, result, rule, selectorTree => {
+
+        selectorTree.walkAttributes(attributeNode => {
+          const attributeSelectorString = attributeNode.toString()
```

Comment:
This might be unnecessary if we're able to use `postcss-selector-parser`'s [`attribute.quoted`](https://github.com/postcss/postcss-selector-parser/blob/master/API.md#attributequoted).